service_identity/src/main/kotlin/org/thechance/service_identity/data/MongoDataBase.kt

```diff
@@ -0,0 +1,12 @@
+package org.thechance.service_identity.data
+
+import com.mongodb.reactivestreams.client.MongoClient
+import org.koin.core.annotation.Single
+import org.litote.kmongo.coroutine.CoroutineDatabase
+import org.litote.kmongo.coroutine.coroutine
+import org.thechance.service_identity.utils.Constants.DATA_BASE_NAME
+
+@Single
+class MongoDataBase(client : MongoClient) {
```

Comment:
For consistency in naming, it would be better renaming MongoDataBase to DataBaseContainer.